cypress/integration/creatorOnboardingFlows/creatorSettings.spec.js

```diff
@@ -120,4 +89,81 @@ describe('Creator Settings Page', () => {
     cy.findByRole('button', { name: 'Finish' }).click();
     cy.url().should('equal', `${baseUrl}admin/creator_settings/new`);
   });
+
+  context('color contrast ratios', () => {
+    it('should show an error when the constrast ratio of a brand color is too low', () => {
+      const lowContrastColor = '#a6e8a6';
+
+      cy.findByLabelText(/^Brand color/)
+        .clear()
+        .type(lowContrastColor)
+        .trigger('change');
+
+      cy.findByText(
+        /^The selected color must be darker for accessibility purposes./,
+      ).should('be.visible');
+    });
+
+    it('should not show an error when the constrast ratio of a brand color is good', () => {
+      const adequateContrastColor = '#25544b';
+
+      cy.findByLabelText(/^Brand color/)
+        .clear()
+        .type(adequateContrastColor)
+        .trigger('change');
+
+      cy.findByText(
+        /^The selected color must be darker for accessibility purposes./,
+      ).should('not.be.visible');
+    });
+  });
+
+  context('brand color updates', () => {
+    it('should not update the brand color if the color contrast ratio is low', () => {
+      const lowContrastColor = '#25544b';
```

Comment:
This looks to be incorrect, on line 139 we use the same value but consider it adequate contrast.

Reply:
✅ Fixed as per Suzannes suggestions below. It seems the async behaviour may have allowed this test to pass, however, I have now validated the error as well the color change for robustness.